content_scripts/vimium_frontend.coffee

```diff
@@ -463,13 +463,7 @@ extend window,
 KeydownEvents =
   handledEvents: {}
 
-  stringify: (event) ->
-    JSON.stringify
-      metaKey: event.metaKey
-      altKey: event.altKey
-      ctrlKey: event.ctrlKey
-      keyIdentifier: event.keyIdentifier
-      keyCode: event.keyCode
+  stringify: (event) -> event.keyCode
```

Comment:
@gdh1995.
Looks like the name `stringify` is no longer appropriate here, right?